src/Drips.sol

```diff
@@ -705,6 +738,35 @@ library Drips {
         return prev.config.lt(next.config);
     }
 
+    /// @notice Calculates the amount dripped over a time range.
+    /// @param cycleSecs The cycle length in seconds.
+    /// Must be the same in all calls working on a single storage instance. Must be higher than 1.
+    /// @param amtPerSec The dripping rate
+    /// @param start The dripping start time
+    /// @param end The dripping end time
+    /// @param amt The dripped amount
+    function _drippedAmt(
+        uint256 cycleSecs,
+        uint256 amtPerSec,
```

Comment:
I agree, it's a huge overkill. It increases the API surface and introduces a new `struct` only to save the user from a single multiplication by a constant. The user also can't be unaware of the multiplication, because the events always hold the multiplied representation.